spec/controllers/application_controller_spec.rb

```diff
@@ -122,4 +122,12 @@ def index
       end
     end
   end
+
+  describe '#analytics' do
+    it 'uses the Analytics class by default' do
+      expect(Analytics).to receive(:new).with(controller.current_user, request)
```

Comment:
Isn't specifying the precise argument making this test more brittle than it needs to be?